tests/test.utils.ts

```diff
@@ -64,13 +64,17 @@ export const uploadManuscript = async (submissionId: string): Promise<AxiosRespo
         variables: {
             id: submissionId,
             file: null,
-            fileSize: 2,
+            fileSize: 115,
         },
     };
 
     body.append('operations', JSON.stringify(operations));
     body.append('map', '{ "1": ["variables.file"] }');
-    body.append('1', 'a', { filename: 'a.txt' });
+    body.append(
+        '1',
+        'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+        { filename: 'a.txt' },
+    );
```

Comment:
Is there an integration test that tests the progress upload sends the expected results?

Reply:
yep. In wizard.files.test.ts